### ISSUE TITLE
fix typescript def for handsontable

### ISSUE THREAD

**Issue by maartenbreddels** [QuantStack/ipysheet#46]
The typescript defs for handsontable are not complete and we get this error:
```bash
src/renderer.ts:25:32 - error TS2339: Property 'registerRenderer' does not exist on type 'Renderers'.

25         Handsontable.renderers.registerRenderer(this.get('name'), this.fn(Handsontable));
```

We can live with it because we ignore errors for now:
https://github.com/QuantStack/ipysheet/blob/36f1ffffb3c564d73174f5c56dfd04bfbcd1dafa/js/tsconfig.json#L6
and we just continue whatever the exit code is:
https://github.com/QuantStack/ipysheet/blob/36f1ffffb3c564d73174f5c56dfd04bfbcd1dafa/js/package.json#L19

We should fix this, preferable by doing a PR to handsontable, and changing the above to be more strict.

**Comment by martinRenou** [comment]
As a workaround, we could put an `// @ts-ignore` before failing lines, in order to prevent build failures.

**Comment by maartenbreddels** [comment]
Fixed in #54
